install/kubernetes/cilium/templates/hubble/tls-cronjob/ca-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if and (or .Values.agent .Values.hubble.relay.enabled .Values.hubble.ui.enabled) .Values.hubble.enabled .Values.hubble.tls.enabled .Values.hubble.tls.auto.enabled (eq .Values.hubble.tls.auto.method "cronJob") }}
+{{- if .Values.hubble.tls.ca.cert }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hubble-ca-secret
+  namespace: {{ .Release.Namespace }}
+type: kubernetes.io/tls
```

Comment:
This line needs to be removed, as this secret unfortunately does not conform to the `tls` type. I got the following error while testing some upgrade paths on this branch:

```
Error: UPGRADE FAILED: cannot patch "hubble-ca-secret" with kind Secret: Secret "hubble-ca-secret" is invalid: [type: Invalid value: "kubernetes.io/tls": field is immutable, data[tls.crt]: Required value, data[tls.key]: Required value]
```

Reply:
My bad, it's updated.